src/main/java/raisetech/StudentManagement/StudentRepository.java

```diff
@@ -1,31 +1,25 @@
 package raisetech.StudentManagement;
 
 import java.util.List;
-import org.apache.ibatis.annotations.Delete;
-import org.apache.ibatis.annotations.Insert;
 import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Result;
+import org.apache.ibatis.annotations.Results;
 import org.apache.ibatis.annotations.Select;
-import org.apache.ibatis.annotations.Update;
 
 @Mapper
 
 public interface StudentRepository {
 
-  @Select("SELECT * FROM student WHERE name = #{name}")
-  Student searchByName(String name);
-
-  @Select("SELECT * FROM student WHERE name = #{name} AND age =#{age}")
-  List<Student> ListStudent(String name,int age);
-
-  @Select("SELECT name, age FROM student")
-  List<Student> ListALLStudents();
-
-  @Insert("INSERT student values(#{name}, #{age})")
-  void registerStudent(String name, int age);
-
-  @Update("UPDATE student SET age = #{age} WHERE name =#{name}")
-  void updateStudent(String name,int age);
-
-  @Delete("DELETE FROM student WHERE name = #{name}")
-  void deleteStudent(String name);
+  @Select("SELECT * FROM student")
+  List<Student> search();
+
+  @Select("SELECT course_id, student_id, course_name, startday, endday FROM student_course")
+  @Results({
+      @Result(property = "courseId", column = "course_id"),
+      @Result(property = "studentId", column = "student_id"),
+      @Result(property = "courseName", column = "course_name"),
+      @Result(property = "startday", column = "startday", javaType = java.time.LocalDateTime.class),
+      @Result(property = "endday", column = "endday", javaType = java.time.LocalDateTime.class)
+  })
+  List<Student_course> course_search();
```

Comment:
メソッド名もキャメルケースにしましょう。
```suggestion
  List<Student_course> courseSearch();
```